cli/cli.go

```diff
@@ -27,6 +27,7 @@ type Flags struct {
 	ResultFile              string
 	ChrootPath              string
 	Database                string
+	Cassandra               string
```

Comment:
Maybe we could also validate in the validation function here that only one of the two databases are set (and add a test for it).
Or we can support specifying two databases at the same time (see comment further down)